src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -2210,40 +2186,5 @@ public SerializerMethods(DeepCopier deepCopy, Serializer serialize, Deserializer
             public Serializer Serialize { get; private set; }
             public Deserializer Deserialize { get; private set; }
         }
-
-        public static JsonSerializerSettings GetDefaultJsonSerializerSettings()
```

Comment:
Don't we still need this for other usages of `OrleansJsonSerializer`, e.g. by storage providers?